front/src/styles/scss/applications/operationalStudies/_nge.scss

```diff
@@ -1,5 +1,9 @@
 .nge-iframe-container {
   width: 100%;
   height: 100%;
-  border: 1px solid pink;
```

Comment:
Oh no, my beloved pink border 🩷

It survived this long, and I will forever remember you as the one who murdered it.